.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
             ${{ runner.OS }}-node-
             ${{ runner.OS }}-
       - name: Install dependencies
-        run: npm ci
```

Comment:
what is the reasoning here: why remove package-lock and not use npm ci?

Reply:
I know @paed01 wants to remove the package-lock file from our open source libs but he probably has a better explanation since I can't remember it